### PR TITLE
[Blockbase] Cleanup + CSS restructuring

### PR DESCRIPTION
**Pull request by kjellr**
This PR adds a readme with basic documentation for Blockbase. Also a little copyediting. 

It also does a little reorganization to the CSS files: 

- The CSS files are split up into their own `front-end` and `editor` folders for clarity.
- There are new `style-custom.css` and `editor-custom.css` files for people to use for theme-specific overrides. This should allow most of the theme CSS to stay untouched, allowing for most structural CSS files to be replaced over time if blockbase were to be updated.
- Cleaned up CSS file headers with descriptions of what each file does.

### CODE REVIEW

File: blockbase/css/front-end/style-base.css [hunk 1]
```diff
@@ -0,0 +1,436 @@
...
+iframe,
+object {
+	max-width: 100%;
+}
+
+/*--------------------------------------------------------------
+# Custom Styles
+--------------------------------------------------------------*/
+
+@import url( "./css/custom.css" );
```
Comment (jffng) [inline blockbase/css/front-end/style-base.css]: I think this import is no longer needed given how you've restructured.

File: blockbase/css/front-end/style-blocks.css [hunk 2]
```diff
@@ -1,3 +1,10 @@
+/*--------------------------------------------------------------
+# Base Block Styles
+#
+# These provide a baseline for front-end block styles. 
+# Please override these in custom.css.
```
Comment (jffng) [inline blockbase/css/front-end/style-blocks.css]:
```suggestion
# Please override these in style-custom.css.
```

File: blockbase/css/front-end/style-base.css [hunk 3]
```diff
@@ -0,0 +1,436 @@
+/*--------------------------------------------------------------
+# Base Styles
+#
+# These provide a baseline for all front-end styles. 
+# Please override these in custom.css.
```
Comment (jffng) [inline blockbase/css/front-end/style-base.css]:
```suggestion
# Please override these in style-custom.css.
```

File: blockbase/css/front-end/style-base.css (hunk 1 above)
Comment (kjellr) [reply blockbase/css/front-end/style-base.css]: Good catch! Should be all set now.

**Pull request merged**